cmd/anonymizer/app/flags.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Options represent configurable parameters for jaeger-anonymizer
+type Options struct {
+	QueryGRPCPort    int
+	QueryGRPCHost    string
+	MaxSpansCount    int
+	TraceID          string
+	OutputDir        string
+	HashStandardTags bool
+	HashCustomTags   bool
+	HashLogs         bool
+	HashProcess      bool
+}
+
+const (
+	queryGRPCHostFlag    = "query-host"
+	queryGRPCPortFlag    = "query-port"
```

Comment:
nit: usually in Jaeger binaries we use host-port flags that accept "host:port" strings.